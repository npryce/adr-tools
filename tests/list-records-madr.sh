adr list
adr init docs madr
adr list
adr new first
adr list
adr new second
adr list
