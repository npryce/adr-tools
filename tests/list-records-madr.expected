adr list
The doc/adr directory does not exist
adr init docs madr
docs/0000-record-architecture-decisions.md
adr list
docs/0000-record-architecture-decisions.md
adr new first
docs/0001-first.md
adr list
docs/0000-record-architecture-decisions.md
docs/0001-first.md
adr new second
docs/0002-second.md
adr list
docs/0000-record-architecture-decisions.md
docs/0001-first.md
docs/0002-second.md
